### ISSUE TITLE
How to reenable sidebar tabs after disabling them?

### ISSUE THREAD

**Issue by SandyRodgers-2017** [MrAlex94/Waterfox#3626]
I disable side bar tabs after install tree style tabs. I have since uninstalled tree style tabs. How do i reenable watefox's default side bar tabs. I can't find a button or option in settings. If it is in about:config I don't know what to change to get the button to come back. Could you help and maybe introduce an option that doesn't dissapear in settings.

**Comment by thunderstone135** [comment]
Look for **_browser.sidebar.disabled_** in $\color{deepskyblue}{about:config}$ and set it to `false`.

**Comment by SandyRodgers-2017** [comment]
> Look for **_browser.sidebar.disabled_** in a b o u t : c o n f i g and set it to `false`.

Tried that didn't work the browser side bar was still disabled. What did  work was going to 'extensions.json' in the profile directory '~/.watefox/ *default*' and opening it. After that, I searched for sidebar@waterfox.net and changed the 'hidden' tag from true to false. After which, I closed the file and loaded back up the browser. After I did this, the sidebar extension now appeared in addons and themes section. From there I reenabled the extension.
